telegram-bot/src/main/java/pro/sky/telegrambot/service/WritingToTheDatabaseImpl.java

```diff
@@ -0,0 +1,39 @@
+package pro.sky.telegrambot.service;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import pro.sky.telegrambot.listener.TelegramBotUpdatesListener;
+import pro.sky.telegrambot.repository.NotificationTaskRepository;
+
+
+@Service
+@NoArgsConstructor
+public class WritingToTheDatabaseImpl implements WritingToTheDatabase {
+    @Getter
+    @Autowired
+    private NotificationTaskRepository repository;
+    @Autowired
+    private MessageProcessingImpl messageProcessing;
```

Comment:
Зависимости строим на абстракциях (интерфейсах)
```suggestion
    private MessageProcessing messageProcessing;
```